04_numeronym/hsy3418/main_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestNumeronym(t *testing.T) {
+
+	actual := numeronyms("accessibility", "Kubernetes", "abc")
+	expected := []string{"a11y", "K8s", "abc"}
```

Comment:
Additional tests are required beyond the ones mentioned in the README... include some tests with spaces and other values so that you can be sure your function works as you expect with other non-standard inputs.  It will be useful to create your additional tests as 'table driven' tests as suggested in the following link...
https://github.com/anz-bank/go-course/wiki/Lab-2:-Collected-Code-Review-Comments